### ISSUE TITLE
Get parsed document as markdown

### ISSUE THREAD

**Issue by greydanus** [deepdoctection/deepdoctection#348]
Is it possible to get the fully-parsed document as a markdown file? This could include titles, bulleted text, and tables. This feature is supported by [LlamaIndex](https://ts.llamaindex.ai/modules/data_loader) and it's quite useful.

**Comment by JaMe76** [comment]
It's not supported out of the box, but generating a markdown format should not be too difficult, I think.

```python
import deepdoctection as dd

analyzer = dd.get_dd_analyzer()

df = analyzer.analyze(path="path/to/dir_or_pdf")
df.reset_state()

mk_str =""
for dp in df:
    for layout in dp.layouts:
        if layout.category_name in ['title']:
            mk_str+= # some conversion using layout.text, layout.category_name
       if layout.category_name in ['text', 'list']:
           # something special for list and text items
    for table in dp.tables:
          # table to markdown
```

This approach can also be used to create a big markdown file for multi page documents.

I am thinking about adding this feature to the `Page` object.

**Comment by greydanus** [comment]
Yes, that's a good point. I ended up building something like that for my purposes - will share the full code when it's working properly.

**Comment by jspast** [comment]
Hi, hate to be that guy, but any update on this? Thanks in advance!

**Comment by jspast** [comment]
Sure, that is completely reasonable. I implemented a simple but not ideal solution for what I need. The only major thing I couldn't do is getting text from `LayoutType.List` segments. Thanks for your response and this great library!
